### ISSUE TITLE
Feature Request: Expose all members of high-level objects

### ISSUE THREAD

**Issue by jo11he** [tudat-team/tudatpy#57]
I notice that the exposition of members of high-level objects is still quite incomplete. This leads to situations where the user is left clueless, e.g. when trying to retrieve environment parameters after a simulation (or estimation) from the environment. 
One example I can give is the `body` object, which does for example not have its `radiation_pressure_interface` member exposed. I am sure there are many more instances of this.

There are more issues with this, namely that there is no `RadiationPressureInterface` class exposed in python, such that - even if the class was exposed as a member of `body` - the members of `RadiationPressureInterface` (or rather the parameters defining the instance of that class) are not accessible to the tudatpy user. 
A good example of this is the `aerodynamic_coefficient_interface`, which is given as a member of the `body` class, but the `AerodynamicCoefficientInterface` class itself is inaccessible to the tudatpy user because it has not yet been exposed.

So in short: If you want to improve tudatpy and the docs that come with it (nicely linking all the connections between objects and their members), a makeover from this "completeness" point of view could be a good start.

**Comment by jo11he** [comment]
There are more issues with this, namely that there is no `RadiationPressureInterface` class exposed in python, such that - even if the class was exposed as a member of `body` - the members of `RadiationPressureInterface` (or rather the parameters defining the instance of that class) are not accessible to the tudatpy user. 

Another example of this is the `aerodynamic_coefficient_interface`, which is given as a member of the `body` class, but the `AerodynamicCoefficientInterface` class itself is inaccessible to the tudatpy user because it has not yet been exposed.

**Comment by gaffarelj** [comment]
I can take care of this, little by little.

I do encourage anyone to drop new elements in this thread when they see that something is still missing, using checkboxes for each element to be exposed so that I can follow whether something has been done already or not.

(cc @DominicDirkx @FilippoOggionni @geoffreygarrett)
